src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -182,16 +182,9 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
             const stopWatch = new StopWatch();
 
             // Fetch the list of kernels ignoring the cache.
-            Promise.all([
-                this.loadNotebookControllersImpl({
-                    listLocalNonPythonKernels: true,
-                    useCache: 'ignoreCache'
-                }),
-                this.loadNotebookControllersImpl({
-                    listLocalNonPythonKernels: false,
-                    useCache: 'ignoreCache'
```

Comment:
We always retrieve both python & non-python kernel specs (hence no need to retrieve non-python kernel specs separately)